### PR TITLE
Improve error message for failed wptserve

### PR DESCRIPTION
**Pull request by lukebjerring**
Bad error messages were highligted by https://github.com/w3c/web-platform-tests/issues/10388

This change now logs a warning as to the exact command that runs (including the config file), _and_ the URL that did not load (in case it's a host file or similar).

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10389)

### CODE REVIEW

**Comment by gsnedders** [comment]
@lukebjerring can you rebase this now master is fixed?

File: resources/test/wptserver.py [hunk 1]
```diff
@@ -22,23 +23,27 @@ def __init__(self, wpt_root):
 
     def start(self):
         self.devnull = open(os.devnull, 'w')
+        wptserveCmd = [os.path.join(self.wpt_root, 'wpt'), 'serve', '--config=' + _CONFIG_FILE]
```
Comment (jgraham) [inline resources/test/wptserver.py]: Note that we use `snake_case`

**Review by jgraham — DISMISSED** [review]

**Comment by lukebjerring** [comment]
Re-synced + snake_cased. PTAL.

**Review by Hexcles — APPROVED** [review]
LGTM except a nit (another snake_case)

File: resources/test/wptserver.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import json
```
Comment (Hexcles) [inline resources/test/wptserver.py]: Where's `json` used?

**Review by Hexcles — COMMENTED** [review]
I was gonna say "let's rebase and land it", but I found one more nit. 

@lukebjerring could you address the comment and merge conflicts?

**Comment by Hexcles** [comment]
@jgraham can you r+? The change you requested has been addressed.

**Pull request merged**